### PR TITLE
feat: support Schema\Builder::getAllTables()

### PR DESCRIPTION
**Pull request by taka-oyama**

### CODE REVIEW

File: src/Query/Processor.php [hunk 1]
```diff
@@ -42,6 +42,18 @@ public function processSelect(Builder $query, $results): array
...
     }
 
+    /**
+     * @param list<array{ TABLE_NAME: string }> $results
+     * @return list<string>
+     */
+    public function processGetAllTables(array $results): array
+    {
+        return array_map(
+            static fn(array $result) => $result['TABLE_NAME'],
```
Comment (tyabu12) [inline src/Query/Processor.php:52]:
```suggestion
            static fn(array $result): string => $result['TABLE_NAME'],
```

File: src/Schema/Grammar.php [hunk 2]
```diff
@@ -48,6 +48,14 @@ public function compileTableExists()
         return 'select * from information_schema.tables where table_schema = \'\' and table_name = ?';
     }
 
+    /**
+     * @return string
+     */
+    public function compileGetAllTables()
+    {
+        return 'select * from information_schema.tables where table_schema = \'\'';
```
Comment (tyabu12) [inline src/Schema/Grammar.php:56]: This SQL includes view table.  

I think it should be to add an additional table_type condition to exclude it as well as [MySqlGrammer::compileGetAllTables()](https://github.com/laravel/framework/blob/9.x/src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php#L60-L68).

https://cloud.google.com/spanner/docs/information-schema#tables

```suggestion
        return 'select * from information_schema.tables where table_schema = \'\' and table_type = \'BASE TABLE\'';
```

**Review by tyabu12 — CHANGES_REQUESTED** [review]

File: src/Query/Processor.php (hunk 1 above)
Comment (taka-oyama) [reply src/Query/Processor.php:52]: I usually don't add return types since phpstan should be taking care of type checking.

**Review by tyabu12 — CHANGES_REQUESTED** [review]
Would you add about `Schema\Builder::getAllTables()` support to CHANGELOG.md?

**Review by tyabu12 — APPROVED** [review]
LGTM

**Pull request merged**